### ISSUE TITLE
USWDS - Bug: Multiselectable Accordion not passing 508

### ISSUE THREAD

**Issue by brunerae** [uswds/uswds#4865]
### Describe the bug

I was trying to implement a multiselectable accordion using uswds components. As directed on the site I added aria-multiselectable="true" attribute to usa-accordion. I have the accordion working as intended. However, I got confirmed 508 violation when scanning the page using the AMP accessibility tool.  The violation reads 'The aria-multiselectable attribute is not allowed on this DIV'.  

### Steps to reproduce the bug

1. add aria-multiselectable="true" attribute to usa-accordion
2. scan using AMP

### Expected Behavior

No 508 errors are recorded by AMP.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.

**Comment by amyleadem** [comment]
Notes from @mejiaj:
" I think our guidance is wrong. MDN talks about aria-multiselectable:
Lists and trees are examples of roles that might allow users to select more than one item at a time.
[aria-multiselectable - Accessibility | MDN (mozilla.org)](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-multiselectable)

Neither of our accordions have the role of grid, listbox, tablist, or tree. And I don't think they have aria-selected on each accordion item either.

So we either need to
- Remove the guidance
- Update the component with the proper markup.

FWIW there are other multiselectable accordions that don't use this aria-attribute
- https://dequeuniversity.com/library/aria/tabless-multiselect-accordion
- https://www.w3.org/WAI/ARIA/apg/example-index/accordion/accordion "
